FlorenceBML/BMLacq776.xml

```diff
@@ -30,10 +31,731 @@ schematypens="http://relaxng.org/ns/structure/1.0"
+                                        <locus from="10v" to="11va"/>
+                                        <title ref="LIT1295Dersan#MiracleHedar"/>
+                                        <incipit xml:lang="gez"><hi rend="rubric">በስመአብ፡ ወወልድ፡ ወመንፈስ፡ ቅዱስ፡ ፩</hi>አምላክ። ንቀድም፡ በረዴዓተ፡ 
+                                            እ<pb n="11ra"/><hi rend="ligature">ግዚ</hi>አብሔር፡ ጽሒፈ፡ ተአምሪሁ፡ ለቅዱስ፡ <gap reason="omitted"/> ሊቀ፡ መላእክት፡ 
+                                            ትንብልናሁ፡ የሀሉ፡ ምስለ፡ ገብሩ፡ ለዓለ፡ ዓለ፡ ዓ፡ <gap reason="omitted"/> ውእቱ፡ እ<hi rend="ligature">ግዚ</hi>አብሔር፡ ስብሕ፡ 
+                                            ወጥቀ፡ ልዑል፡ ዲበ፡ ኩሉ፡ ፍጥረት፡ ዝኢያገምሮ፡ ህሊና፡ ሰብእ፡ ወኢአምሮ፡ ዘመላእክት፡ ለፈጣሬ፡ ሰማያት፡ ወምድር፡ እስመ፡ ውእቱ፡ አዘዘ፡ በቃለ፡ ጥበቡ፡ 
+                                            ወአቀሞሙ፡ ለዓለም፡ ትእዝዝ፡ ወሀቦሙ፡ ወኢሃለፉ፡ </incipit>
+                                        <explicit xml:lang="gez">ወለእለ፡ ይትለአከዎ፡ ነደ፡ እሳት፡ ስልስ፡ ዲቤ፡ ዝያ፡ ትአቦን፡ መልአከ፡ እ<hi rend="ligature">ግዚ</hi>አብሔር፡ 
+                                            አውዶሙ፡ ለእለ፡ ይሪፈርህዎ፡ ወያድኅኖሙ፡ ጠአሙ፡ ወታእምሩ፡ ከመ፡ ሄር፡ እ<hi rend="ligature">ግዚ</hi>አብሔር፡ ወመንከር፡ ጥበብከ፡ 
+                                            <pb n="11va"/>እ<hi rend="ligature">ግዚ</hi>ኦ፡ ስብሐት፡ ለብ፡ ስብሐት፡ ለወልድ፡ ስብሐት፡ ለመንፈስ፡ ቅዱስ፡ ይዜኒ፡ ወዘልፈኒ፡ ወለአለመ፡ 
+                                            ዓ፡ ዓ፡ </explicit>
+                                    </msItem>
+                                    <msItem xml:id="ms_i1.1.3">
+                                        <locus from="11va" to="12rb"/>
+                                        <title ref="LIT1295Dersan#CommemorativeHedar"/>
+                                        <incipit xml:lang="gez"><hi rend="rubric">በስመአብ፡ ወወልድ፡ ወመንፈስ፡ ቅዱስ፡ </hi>፩አምላክ፡ በዘቲ፡ እለት፡ አ፲፪ለሕዳር፡ ተዝካረ፡ በአሉ፡ 
+                                            ለመላክ፡ ክቡር፡ <hi rend="rubric">ሚካኤል፡ ሊ</hi>ቀ፡ መላእክት፡ ወርእሰ፡ ሰማያውያን፡ ወምድራውያን፡ መላእክት፡ መልአክ፡ መሐሪ፡ ለዘመደ፡ 
+                                            እጓለ፡ እመሕያው፡ ዝይቅውምም፡ ኩሎ፡ ጊዜ፡ ቅድመ፡ መንበረ፡ እባዩ፡ ለአብ፡ ይትነበል፡ በእንተ፡ ኩሉ፡ ፍጥረት፡ ዝንቱ፡ ዘአርአዩ፡ ለኢያሱ፡ </incipit>
+                                        <explicit xml:lang="gez">እስከ፡ ይፌጽሙ፡ ገድሎሙ፡ ወተገብረ፡ ሎቱ፡ ተዝካረ፡ ባወምጽዋታት፡ በእንተ፡ ስሙ፡ በበ፲ወ፪ቱ፡ ወርኅ፡ እስመ፡ ውእቱ፡ ይስእሎ፡ 
+                                            ለእ<hi rend="ligature">ግዚ</hi>አብሔር፡ በእንተ፡ ፍሬያተ፡ ምድር፡ ወበእንተ፡ ማያተ፡ አፍላግ፡ ወበእንተ፡ ነፍሳት፡ ሠናያን፡ ከመ፡ ይፈጽሞሙ፡ 
+                                            እ<hi rend="ligature">ግዚ</hi>አብሔር፡ ለኩሎሙ፡ በርትሰ፡ ትንብልናሁ፡ ያሃሉ፡ ምስ፡ ገብ<del rend="strikethrough">ሩ</del>ሩ፡ 
+                                            <gap reason="omitted"/> ለዓለመ፡ ዓሜን፡ </explicit>
+                                    </msItem>
+                                    <msItem xml:id="ms_i1.1.4">
+                                        <locus from="12rb" to="13rb"></locus>
+                                        <title ref="LIT1295Dersan#MiracleHedar"></title>
+                                        <note>Attested also in <ref target="#EMIP00016"/> on f. 9v to 13r.</note>
```

Comment:
`<ref type="mss" corresp="EMIP00016"/>` would be correct here. `<ref target="#"/>` is only used to refer to xml:ids within the same file. I know it's confusing, it comes with time

Reply:
Done!

Reply:
also in line 458.